tests/gate/test_review_settled.py

```
@@ -84,6 +91,29 @@ def test_human_opened_threads_are_ignored(self) -> None:
+        rng = f"between e60004682df7de66be55e0acd83445281d8c421c and {head}"
+        walkthrough = {"user": {"login": "coderabbitai[bot]"},
+                       "body": f"<!-- auto-generated comment: summarize by coderabbit.ai -->\nReviewing files {rng}."}
+        limited = {"user": {"login": "coderabbitai[bot]"},
+                   "body": f"<!-- auto-generated comment: summarize by coderabbit.ai -->\n<!-- auto-generated comment: rate limited by coderabbit.ai -->\nReview limit reached. {rng}."}
+        pending = {"user": {"login": "coderabbitai[bot]"},
+                   "body": f"<!-- auto-generated comment: summarize by coderabbit.ai -->\n<!-- auto-generated comment: review in progress by coderabbit.ai -->\nCurrently processing {rng}."}
+        f = reviewed_filter()
+        def count(comments):
+            # The workflow feeds `--paginate` output (one JSON array per page) to jq -s,
+            # so the input is one page array; -s wraps it into the outer list itself.
+            out = subprocess.run(["jq", "-s", "--arg", "head", head, f],
```

Ruff (0.16.3)

[error] `subprocess` call: check for execution of untrusted input

(S603)